tendrl/monitoring_integration/manager/__init__.py

```diff
@@ -69,8 +69,8 @@ def _upload_default_dashboards():
                            {'message': msg})
             else:
                 msg = '\n' + "Dashboard " + str(dashboard_json) + \
-                   " upload failed with error code " + str(response.status_code) + \
-                   '\n'
+                   " upload failed with Error Code " + str(response.status_code) + \
```

Comment:
Couldnt it be just `Dashboard {name} update failed. Error Code: {code}. Error: {msg}`